### ISSUE TITLE
Not all data coming through: Notification details

### ISSUE THREAD

**Issue by Tycho-S** [rootkiwi/an2linuxclient#38]
I don't see the full content for all notifications. For example, the ones from Samsung email have 3 components when I see them as a notification on the phone:

- Sender (the title in an2linux)
- The Subject (the message in an2linux)
- The first few lines of the email. This is the part that's not coming through.

With other notifications it works similarly though what's in each of the 3 fields can be different of course. Not all of them have the third detail field.
 
When it's there I can see this third field when expand the notification on the phone. I also see this on my watch and smart band (using gadgetbridge) so forwarding this info should be possible. It would be great if this could be added too.

**Comment by rootkiwi** [comment]
Yeah that should be possible by extracting: https://developer.android.com/reference/android/app/Notification#EXTRA_TEXT_LINES

Will have to test it, and if it works add it to the app soon.

**Comment by Tycho-S** [comment]
Thank you for looking into this! Sorry I didn't reply for so long as I was working on other things (mainly work related). But it would really be great to get all the info. It would also be really nice to get the name of the sending app (without replacing the subject) any and other data :) So I can group them on the notification panels. Thanks for considering this.
